tests/endtoend/blob_functions/get_blob_bytes/function.json

```diff
@@ -16,7 +16,7 @@
     {
       "type": "http",
       "direction": "out",
-      "name": "$return",
```

Comment:
Nice to fix but I prefer to not tweak parts of the code that we do not touch.

Reply:
This is actually needed - in their mock worker, when they load an app (i.e. load all functions in this directory) it parses all the json files and breaks if the json files are not in the right format (apparently dangling commas are invalid json syntax).

Reply:
Can you in parallel do a PR with this change and if it gets merged before this, we rebase this one?
Easier to track with blame later.